docs/en/benchmark/performance_evaluation_guide.md

```diff
@@ -82,67 +82,67 @@ The next two performance observation and analysis tools are essential tools for
-  - txn/lat: the number of **write transactions** processed by the metadata engine per second and their average latency (in milliseconds). Read-only requests such as `getattr` are only counted as ops and not txn.
-  - retry: the number of **write transactions** that the metadata engine retries per second
+  - ops/lat: requests per second processed by the metadata engine and their average latency (in milliseconds). Please note that some requests that can be processed directly in the cache are not included in the statistics, in order to better reflect the time spent by the client interacting with the metadata engine.
+  - txn/lat: **write transactions** per second processed by the metadata engine and their average latency (in milliseconds). Read-only requests such as `getattr` are only counted as ops but not txn.
+  - retry: **write transactions** per second that the metadata engine retries
 - blockcache
-  - read/write: read/write traffic per second for the client's local data cache
+  - read/write: read/write traffic per second for the local data cache of the client
 - object
-  - get/get_c/lat: bandwidth value of object store per second for processing **read requests**, number of requests and their average latency (in milliseconds)
-  - put/put_c/lat: bandwidth value of object store for **write requests** per second, number of requests and their average latency (in milliseconds)
-  - del_c/lat: the number of **delete requests** per second and the average latency (in milliseconds) of the object store
+  - get/get_c/lat: bandwidth, requests per second, and their average latency (in milliseconds) for object storage processing **read requests**
+  - put/put_c/lat: bandwidth, requests per second, and their average latency (in milliseconds) for object storage processing **write requests**
+  - del_c/lat: **delete requests** per second the object storage can process, and the average latency (in milliseconds)
 
 ### JuiceFS Profile
 
-JuiceFS `profile` is used to output all access logs of the JuiceFS client in real time, including information about each request. It can also be used to play back and count the JuiceFS access logs, allowing users to visualize the operation of JuiceFS (for detailed description and usage see [documentation](operations_profiling.md)). When executing `juicefs bench`, the following command is executed in another session. When the `juicefs bench` is running, create a new session and execute the following command.
+JuiceFS `profile` is used to output all access logs of the JuiceFS client in real time, including information about each request. It can also be used to play back and count the JuiceFS access logs, allowing users to visualize the running status of JuiceFS (for detailed description and usage. See [documentation](operations_profiling.md)). Run the following command in another session while the `juicefs bench` command is running.
```

Comment:
```suggestion
JuiceFS `profile` is used to output all access logs of the JuiceFS client in real time, including information about each request. It can also be used to play back and count JuiceFS access logs, and visualize the JuiceFS running status (see [documentation](operations_profiling.md) for details). To run the JuiceFS profile, execute the following command in another session while the `juicefs bench` command is running.
```